stable/cf-operator/charts/quarks-job/README.md

```diff
@@ -9,12 +9,43 @@ This helm chart deploys the quarks-job operator.
 To install the latest stable helm chart, with `quarks-job` as the release name into the namespace `quarks`:
 
 ```bash
-$ helm install quarks-job https://s3.amazonaws.com/cf-operators/helm-charts/quarks-job-v0.0.1%2B47.g24492ea.tgz --namespace quarks
+helm repo add quarks https://cloudfoundry-incubator.github.io/quarks-helm/
+helm install quarks-job quarks/quarks-job
 ```
 
+### Using multiple operators
+
+Choose different namespaces and cluster role names. The persist-output service account will be named the same as the cluster role:
+
+```
+helm install relname1 quarks/quarks-job \
+  --namespace namespace1
+  --set "global.singleNamespace.name=staging1" \
+  --set "persistOutputClusterRole.name=clusterrole1" \
+```
+
+### Using multiple namespaces with one operator
+
+The cluster role can be reused between namespaces.
+The service account (and role binding) should be different for each namespace.
+
+```
+helm install relname1 quarks/cf-operator \
+  --set "global.singleNamespace.create=false" \
+  --set "persistOutputClusterRole.name=qjob-persist-output"
+```
+
+Manually create before running `helm install`, for each namespace:
+
+* a namespace "staging1" with the following labels:
+  * quarks.cloudfoundry.org/monitored: relname1
+  * quarks.cloudfoundry.org/qjob-service-account: qjob-account1
+* a service account named "qjob-account1"
+* a role binding from the existing cluster role "qjob-persist-output" to "qjob-account1" in namespace "staging1"
+
 ## Installing the Chart From the Developmenet Branch
```

Comment:
typo: `Development`. Will hunt down the original and update there but OK for now.